packages/react/src/components/List/List.tsx

```diff
@@ -920,14 +952,18 @@ export class List<T = any> extends React.Component<IListProps<T>, IListState<T>>
       const isPageInAllowedRange = !allowedRect || (pageBottom >= allowedRect.top && pageTop <= allowedRect.bottom!);
       const isPageInRequiredRange =
         !this._requiredRect || (pageBottom >= this._requiredRect.top && pageTop <= this._requiredRect.bottom!);
+
+      const isPagePartiallyInRequiredRange =
+        !isPageInRequiredRange &&
+        this._requiredRect &&
+        ((pageTop < this._requiredRect.top && pageBottom > this._requiredRect.top) ||
+          (pageTop < this._requiredRect.bottom! && pageBottom > this._requiredRect.bottom!));
```

Comment:
Why do we need `!` for `bottom` but not for `top`?

Reply:
[Because `bottom` is optional on `IRectangle`](https://github.com/microsoft/fluentui/blob/master/packages/utilities/src/IRectangle.ts#L13).

However, I don't need this code so I'm going to remove it :)